framework/helpers/BaseStringHelper.php

```diff
@@ -338,4 +338,31 @@ public static function base64UrlDecode($input)
     {
         return base64_decode(strtr($input, '-_', '+/'));
     }
+
+    /**
+     * @param string $token An unmasked token.
+     * @return string A masked token.
+     * @since 2.0.13
+     */
+    public static function maskCsrfToken($token)
```

Comment:
Could it be applied to non-CSRF or non-tokens? If yes, the name is too specific. If not, methods are better not exposed at all and kept protected in the request.